awx/main/models/inventory.py

```diff
@@ -2343,16 +2340,26 @@ def inventory_as_dict(self, inventory_update, private_data_dir):
                 if not striped_hf:
                     continue
                 ret['filters'].append(striped_hf)
+        else:
+            # default behavior filters by power state
+            ret['filters'] = ['guest.ipAddress']
```

Comment:
had this wrong, was given wrong number but not realizing it. Pushing next commit.